docker-compose.prod.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.6'
+
+services:
+  app:
+    build: &build
+      context: .
+      dockerfile: docker/app-prod/Dockerfile
+      args:
+        - GID=${H2_GID-1000}
+        - UID=${H2_UID-1000}
+        - SECRET_KEY_BASE
+    environment: &environment
+      - DATABASE_NAME=h2
+      - DATABASE_USERNAME
+      - DATABASE_PASSWORD
+      - DATABASE_HOSTNAME=host.docker.internal
+      # for ActionCable
+      - REDIS_URL=redis://host.docker.internal:6379/
+      - SMTP_HOST=host.docker.internal
+      - HOST
+      - HONEYBADGER_API_KEY
+      - SETTINGS__RABBITMQ__PASSWORD
+      - SETTINGS__RABBITMQ__QUEUES__DEPOSIT_COMPLETE
```

Comment:
Do we need these `*_QUEUES_*` variables in here?  Aren't they basically constants in settings.yml?

Reply:
No, they vary by server so that we can run multiple instances of H2 (e.g., dockerized and non-dockerized) and each has its own queue. This is set in puppet.